components/brave_vpn/resources/panel/components/sell-panel/style.ts

```diff
@@ -4,65 +4,80 @@
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 import styled from 'styled-components'
 import VPNSvgUrl from '../../assets/svg-icons/vpn-icon.svg'
-import checkIconUrl from '../../assets/svg-icons/check-gradient-icon.svg'
+import checkIconUrl from '../../assets/svg-icons/shield-done.svg'
 import sellGraphicUrl from '../../assets/svg-icons/sell-graphic.svg'
 import guardianLogoUrl from '../../assets/svg-icons/guardian-logo.svg'
 
 export const Box = styled.div`
   width: 100%;
   height: 100%;
-  background: #381E85;
+  background: #0F0663;
   position: relative;
 `
 
-export const ProductTitle = styled.h1`
-  color: #F0F2FF;
+export const PanelContent = styled.section`
+  display: flex;
+  padding: var(--leo-spacing-3xl) var(--leo-spacing-2xl);
+  flex-direction: column;
+  align-items: center;
+  gap: var(--leo-spacing-2xl);
+`
+
+export const ProductTitle = styled.h3`
+  color: #FFF;
   font-family: ${(p) => p.theme.fontFamily.heading};
   font-weight: 600;
-  font-size: 20px;
-  margin: 0 0 10px 0;
+  font-size: 22px;
+  font-style: normal;
+  margin: 0 0 var(--leo-spacing-xl) 0;
+  text-align: center;
+  line-height: var(--leo-typography-heading-h3-line-height, 28px);
+  letter-spacing: var(--Letter-spacing-Headings, -0.5px);
 `
 
 export const PoweredBy = styled.div`
-  margin-bottom: 16px;
   display: flex;
   align-items: center;
 
   span {
     font-family: ${(p) => p.theme.fontFamily.heading};
     font-style: normal;
-    font-weight: normal;
-    font-size: 12px;
-    color: #C2C4CF;
+    font-weight: 400;
+    font-size: 14px;
+    color: #FFF;
+    text-align: center;
   }
 `
 
 export const List = styled.ul`
+  align-self: start;
   list-style-type: none;
-  padding: 0 32px;
-  margin: 0 0 16px 0;
+  padding: 0;
+  margin: 0;
 
   li {
-    color: #F0F2FF;
+    color: #FFF;
     font-family: ${(p) => p.theme.fontFamily.heading};
-    font-weight: 600;
+    font-weight: 400;
     font-size: 12px;
+    font-style: normal;
     line-height: 18px;
-    margin-bottom: 8px;
-    padding-left: 26px;
-    text-indent: -26px;
+    margin-bottom: var(--leo-spacing-m);
+    padding-left: 28px;
+    text-indent: -28px;
+    vertical-align: middle;
```

Comment:
Curious why `vertical-align` doesn't work with browser build but works for storybook.
Left one is from browser's vpn panel and right one is from storybook.
Both have different text vertical align in `li`.
@fallaciousreasoning Can you help why I'm seeing different result with same code?
<img width="250" alt="Screenshot 2024-07-15 at 2 55 47 PM" src="https://github.com/user-attachments/assets/6b1757b8-746d-4e34-8c86-e03f9222876e">
<img width="250" alt="Screenshot 2024-07-15 at 2 56 35 PM" src="https://github.com/user-attachments/assets/0b3faba5-e62c-4698-a8da-260345b8bd00">

Reply:
I wonder if it's because we set `
  vertical-align: inherit;
` on `::before` and `::after` in our CSS reset?

Reply:
You should be able to see what's setting the `vertical-align` on the `::before` in dev tools on the `computed` tab

Reply:
There is nothing special about it
![image](https://github.com/user-attachments/assets/ed253ff3-1712-472f-97fb-da1355515f4f)

Reply:
Ah, I set `vertical-algin` to `::before` and above screen shot has `vertical-align` property.

This is from storybook
![image](https://github.com/user-attachments/assets/5f53f7a3-5c97-47a7-b77c-2236f525eef1)

Reply:
Ah, when I set it `li`, it's not inherited to  it's children such as`::before` and text content.
but it's inherited from li to `:;before` or text content in storybook.

Reply:
This emptykit.css made this difference.
![image](https://github.com/user-attachments/assets/4fe860ba-f7cf-4b0c-974a-aec9a8c92a23)

Reply:
Maybe this is more better changes?
```
$ git diff
diff --git a/components/brave_vpn/resources/panel/components/sell-panel/style.ts b/components/brave_vpn/resources/panel/components/sell-panel/style.ts
index 9a484e093be..15a2dae48dd 100644
--- a/components/brave_vpn/resources/panel/components/sell-panel/style.ts
+++ b/components/brave_vpn/resources/panel/components/sell-panel/style.ts
@@ -57,6 +57,7 @@ export const List = styled.ul`
     margin-bottom: var(--leo-spacing-m);
     padding-left: 28px;
     text-indent: -28px;
+    vertical-align: middle;

     &:before {
       content: '';
@@ -69,7 +70,7 @@ export const List = styled.ul`
       user-select: none;
       pointer-events: none;
       margin-right: 8px;
-      vertical-align: middle;
+      vertical-align: inherit;
     }

     &:last-child {
```

Reply:
Yeah, looks good!